packages/react-web-kit/src/components/molecules/Grid/Item/Item.tsx

```diff
@@ -1,94 +1,88 @@
-  flex?: number;
-  flexDirection?: 'row' | 'column' | 'row-reverse' | 'column-reverse';
-  flexGrow?: number;
-  flexShrink?: number;
-  justifyContent?: FlexPositioning | FlexSpacing | 'space-evenly';
-  /** You don't have to give this property since is inherited from Grid */
-  spacing?:
-    | SpacingType
-    | {
-        top?: SpacingType;
-        right?: SpacingType;
-        bottom?: SpacingType;
-        left?: SpacingType;
-      };
-  /** Used to wrap children in a View when its style extrapolates the dimensions*/
-  wrapper?: boolean;
+export interface IGridItemWeb
+  extends Omit<IGridItem, 'span'>,
+    Omit<React.HTMLAttributes<HTMLDivElement>, 'children'> {
+  /** At least sm should exist */
+  span: number | Span;
+  /** Only specify this property if the GridItem will be 'dynamic', adjusting itself to content. Use this with `wrapper` for better result. */
+  flexBasis?:
+    | string
+    | 'content'
+    | 'auto'
+    | 'max-content'
+    | 'min-content'
+    | 'fit-content';
 }
 
 const GridItem = ({
   children,
-  span,
+  span: _span,
   columns = 12,
   loadingComponent,
   loading = false,
   spacing: _spacing,
   wrapper = false,
+  alignContent,
+  alignItems,
+  alignSelf,
+  flex,
+  flexDirection,
+  flexGrow,
+  flexShrink,
+  justifyContent,
+  flexBasis,
+  style,
   ...rest
-}: IGridItem) => {
+}: IGridItemWeb): JSX.Element => {
   const { spacing } = useTheme();
+  const { sm, md, lg } = useBreakpoints();
   if (!React.Children.only(children)) {
     throw new Error('The number of children in GridItem should be one');
   }
   if (loadingComponent && loading) {
     return loadingComponent;
   }
 
-  const getPadding = (pos: PaddingPosition) => {
-    if (_spacing) {
-      if (typeof _spacing === 'string')
-        return extractNumbersFromString(spacing[_spacing]);
-      else if (typeof _spacing === 'object' && _spacing[pos]) {
-        return extractNumbersFromString(spacing[_spacing[pos]!]);
-      } else return undefined;
-    } else return undefined;
-  };
+  const span = typeof _span === 'number' ? _span : getSpan(_span, sm, md, lg);
 
-  const style: React.CSSProperties = {
-    ...rest,
+  const _style: React.CSSProperties = {
+    ...style,
+    alignContent,
+    alignItems,
+    alignSelf,
+    flexDirection,
+    flexGrow,
+    flexShrink,
+    justifyContent,
+    flex,
     boxSizing: 'border-box',
-    flexBasis: `${100 / (columns / span)}%`,
-    paddingTop: getPadding('top'),
-    paddingBottom: getPadding('bottom'),
-    paddingRight: getPadding('right'),
-    paddingLeft: getPadding('left'),
+    flexBasis: flexBasis ?? `${100 / (columns / span)}%`,
+    paddingTop: getPadding('top', _spacing, spacing),
+    paddingBottom: getPadding('bottom', _spacing, spacing),
+    paddingRight: getPadding('right', _spacing, spacing),
+    paddingLeft: getPadding('left', _spacing, spacing),
   };
 
   const clone = React.cloneElement(children, {
     ...children?.props,
```

Comment:
@dennervidal ficou faltando repassar o ...rest quando nao é wrapper

Reply:
quando não for wrapper, as propriedades da grid item não acho uma boa repassar pro clone (filho de grid item), isso pode gerar algum efeito indesejado, até porque o grid item extende View no nativo e div na web, e nem sempre o filho do grid vai ser div/View, então por esse motivo eu preferi preservar o clone e só fazer a passagem pro wrapper.